pages/user/setting.js

```diff
@@ -0,0 +1,110 @@
+import UserHeader from '../components/user_header';
+import SettingNav from '../components/setting_nav';
+
+const chooseFile = () => {
```

Comment:
## Critical
- This need not be outside the component

- Instead of targeting `document` why not target the event object itself
i.e. 
```js
const choseFile = (event) => {
const {value} = event.target 
// do whatever you want with the value....
}
```